src/modules/Admin/Components/Home/Components/Home.tsx

```diff
@@ -1,7 +1,178 @@
-import React from 'react'
+import * as React from 'react';
+import axios from "axios";
+import WorkOutlineIcon from '@mui/icons-material/WorkOutline';
+import { Doughnut } from 'react-chartjs-2';
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js';
+import { toast } from 'react-toastify';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
 
 export default function Home() {
+  const [dashboardData, setDashboardData] = React.useState<any>(null);
+
+  const Submit = async () => {
+    try {
+      const response = await axios.get(`https://upskilling-egypt.com:3000/api/v0/admin/dashboard`, {
```

Comment:
The url shouldn't be hardcoded, this will make it harder to change it if the url changes or if we want to update it.
We can provide the base url and create an axios instance and then use this instance.

Reply:
We need to provide the response type to get autocompletion and errors if we are accessing something wrong.